src/schemas/21-526EZ-allclaims/schema.js

```diff
@@ -594,78 +594,93 @@ const schema = {
-              type: 'string'
-            },
-            incidentDescription: {
-              type: 'string'
-            },
-            unitAssigned: {
-              type: 'string'
-            },
-            unitAssignedDates: {
-              $ref: '#/definitions/dateRange'
-            },
-            remarks: {
-              type: 'string'
-            },
-            additionalChanges: {
-              type: 'string'
-            },
-            personInvolved: {
-              type: 'array',
-              items: {
-                type: 'object',
-                name: {
-                  $ref: '#/definitions/fullName'
-                },
-                rank: {
-                  type: 'string'
-                },
-                injuryDeath: {
-                  type: 'string',
-                  enum: [
-                    'Killed in Action',
-                    'Killed Non-Battle',
-                    'Wounded in Action',
-                    'Injured Non-Battle',
-                    'Other'
-                  ]
-                },
-                injuryDeathOther: {
-                  type: 'string'
-                },
-                injuryDeathDate: {
-                  $ref: '#/definitions/date'
-                },
-                unitAssigned: {
-                  type: 'string'
+      properties: {
+        remarks: {
+          type: 'string'
+        },
+        additionalIncidentText: {
+          type: 'string'
+        },
+        additionalSecondaryIncidentText: {
+          type: 'string'
+        },
+        otherInformation: {
+          type: 'array',
+          items: {
+            type: 'string'
+          }
+        },
+        incident: {
+          type: 'array',
+          minItems: 1,
+          items: {
+            type: 'object',
+            required: [ 
+              'personalAssault' 
+            ],
+            properties: {
+              personalAssault: {
+                type: 'boolean'
+              },
+              medalsCitations: {
+                type: 'string'
+              },
+              incidentDate: {
+                $ref: '#/definitions/date'
+              },
+              incidentLocation: {
+                type: 'string'
+              },
+              incidentDescription: {
+                type: 'string'
+              },
+              unitAssigned: {
+                type: 'string'
+              },
+              unitAssignedDates: {
+                $ref: '#/definitions/dateRange'
+              },
+              personInvolved: {
+                type: 'array',
+                items: {
+                  type: 'object',
+                  name: {
+                    $ref: '#/definitions/fullName'
+                  },
+                  rank: {
+                    type: 'string'
+                  },
+                  injuryDeath: {
+                    type: 'string',
+                    enum: [
+                      'Killed in Action',
```

Comment:
We typically shy away from using nicely-formatted things in enum lists, because it's much more difficult to make content changes down the road, and any content changes you make will affect the api as well. Would recommend sticking to the categories like `['KILLED_ACTION', 'KILLED_NONBATTLE', 'WOUNDED_BATTLE'` etc and then managing the `enumNames` on the front end as needed.